backend/controller/dal/dal.go

```diff
@@ -899,6 +915,102 @@ func (d *DAL) ExpireRunnerClaims(ctx context.Context) (int64, error) {
 	return count, translatePGError(err)
 }
 
+type JobState string
+
+const (
+	JobStateIdle      = JobState(sql.JobStateIdle)
+	JobStateExecuting = JobState(sql.JobStateExecuting)
+)
+
+type CronJob struct {
+	id            int64
```

Comment:
The pattern we've been taking is to create custom key types rather than exposing database IDs through the DAL. Create a new `cronjob_key.go` file and replace this with that. We'll want to expose this to the users anyway, so they can have a usable reference to specific cron jobs, similar to how we do this for deployments etc.

This might simplify some of the DAL methods too, in that the keys could be used to refer to cron jobs rather than a full CronJob type, potentially (depending on how you're using it).